### ISSUE TITLE
Stop using "HIT", call it a "job" or a "task"

### ISSUE THREAD

**Issue by martinpopel** [AppraiseDev/Appraise#191]
In past, each job had a two HITs, which was quite confusing especially for newly hired annotators. Now, each job contains just a single HIT (at least in the WMT campaigns, though I would suggest it everywhere), but Appraise still uses messages such as "Current HIT" or "X annotations, Y HITs completed".

I would prefer using terms "job" or "task" (just one of them and consistently).

I know HIT=Human Intelligence Task, but this is an Amazon MTurk specific term (at least I don't know about any other tool than Appraise and MTurk that would use it) and I doubt an average native English speaker knows this. We use Appraise even for non-English language pairs, where the annotators are not required to speak English. Using terms in the Appraise UI that are difficult to translate using a dictionary (or worse, easy to translate but with a wrong meaning, as is the case of "hit") is not a good idea.

**Comment by zouharvi** [comment]
Agreed! Since Appraise is almost exclusively used for 1 account = 1 task, it would make sense to simplify this both in terms of naming and in code.
This would need a major refactor put could potentially make the codebase healthier.

**Comment by martinpopel** [comment]
OK. My suggestion would be to first just `s/HIT/task/g` in the UI and keep the implementation intact. Second, refactor the implementation. (This issue is not about forbidding multiple tasks for one account/username, that could be still useful in many cases where GDPR is not an obstacle.)

**Comment by zouharvi** [comment]
Resolved in #197.
